### PR TITLE
Fix math load test accept ibm result

### PR DESCRIPTION
**Pull request by Mesbah-Alam**

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
Details of this PR : https://github.com/AdoptOpenJDK/openjdk-systemtest/issues/8

**Comment by Mesbah-Alam** [comment]
Personal build : https://ci.adoptopenjdk.net/view/work%20in%20progress/job/test_personal/178/consoleFull

**Comment by Mesbah-Alam** [comment]
PR : https://github.com/AdoptOpenJDK/openjdk-systemtest/pull/71

File: openjdk.test.math/src/test.math/net/adoptopenjdk/test/math/MathAPITest.java [hunk 1]
```diff
@@ -668,7 +668,8 @@ public void testExp()
 		// At 20170906, IBM Java returns 0.29428694035725067, not 0.29428694035725067
 		// assertEquals("exp(double)[7] ::", 0.2942869403572507, Math.exp(-1.2232D));
 		assertEquals("exp(double)[8] ::", 1.0, Math.exp(0D));
-		assertEquals("exp(double)[9] ::", 11.3842406513381, Math.exp(2.43223D));
+		assertTrue("exp(double)[9] ::", Math.exp(2.43223D) == 11.3842406513381 || Math.exp(2.43223D) == 11.384240651338098);
```
Comment (lumpfish) [inline openjdk.test.math/src/test.math/net/adoptopenjdk/test/math/MathAPITest.java]: Is the commented out line 2 lines above a similar situation?  If so, they may as well get fixed together.  Also, please could you add a comment as to why two values are permissible.  That would be useful to anyone not familiar with the history and discussion.

Comment (Mesbah-Alam) [reply openjdk.test.math/src/test.math/net/adoptopenjdk/test/math/MathAPITest.java]: Yes, I have corrected the other test too and added comments. The change has been tested using personal build here: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/test_personal/181/console

Comment (lumpfish) [reply openjdk.test.math/src/test.math/net/adoptopenjdk/test/math/MathAPITest.java]: Thanks @Mesbah-Alam .  Looks good.

**Pull request merged**